src/function/builtin.rs

```diff
@@ -76,7 +82,22 @@ pub fn builtin_function(identifier: &str) -> Option<Function> {
         "floor" => simple_math!(floor),
         "round" => simple_math!(round),
         "ceil" => simple_math!(ceil),
+        "math::is_nan" => float_is(f64::is_nan),
```

Comment:
Could you add a comment here, something like "// Float special values" or so

Reply:
@ISibboI : Ok ! I added it in [27bd1a3](https://github.com/ISibboI/evalexpr/pull/101/commits/27bd1a32d702e17c3b0aa83106f2f7b7b485067d)